### DISCUSSION TITLE
423037A3-930F-4F6A-BA92-E6E551391BE8

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#73] [General]
# HD 10180 (Unconfirmed Planets)

The three candidate planets of HD 10180, in addition to its six confirmed planets. One of these candidates was identified by the system's discovery paper in 2010 (hence the 

https://celestia.mobi/addon?item=423037A3-930F-4F6A-BA92-E6E551391BE8&lang=en

**Comment by AU-Microscoppi-b** [comment]
HD 10180 b has a missing texture, i and j are fine
